quint/test/parsing/quintParserFrontend.test.ts

```diff
@@ -113,31 +120,104 @@ describe('parsing', () => {
   })
 })
 
-describe('parse errors', () => {
-  it('error in module unit', () => {
-    parseAndCompare('_1002emptyWithError')
+// instead of testing how errors are produced in json,
+// we only test the error messages
+describe('syntax errors', () => {
+  it('unbalanced module definition', () => {
+    const code = 'module empty {'
+    const errors = parseErrorsFrom('module-unbalanced', code)
```

Comment:
Can we make this source name parameter fixed so a value? I don't see why we need to name the source name for every test differently